.github/workflows/ci-core-reusable.yml

```diff
@@ -318,45 +367,6 @@ jobs:
       #     ci_run sleep 10
       #     ci_run zk test i upgrade
 
-      - name: Show server.log logs
```

Comment:
Do we not want to show logs? Probably something else is displaying them

Reply:
yeah, the Upload logs action publishes all the logs to github and can be accessible via a link (very helpful when we'll run tests for multiple chains at the same time)